testnet/stacks-node/Cargo.toml

```diff
@@ -16,6 +16,10 @@ serde_json = { version = "1.0", features = ["arbitrary_precision", "raw_value"]
 stacks = { package = "blockstack-core", path = "../../." }
 toml = "0.5.6"
 
+[dev-dependencies]
+warp = "0.2"
+tokio = "0.2.21"
```

Comment:
Why?  Isn't tokio at least already included?  Also, what's warp?

Reply:
WARP is a webserver built with hyper -- it's used only to set up a testing sidecar. The tokio runtime is explicitly required to start up this testing webserver. While tokio is an upstream dependency of other things included here, we need it as an explicit dependency to start the tokio runtime.